baremetal_network_provisioning/ml2/mechanism_hp.py

```diff
@@ -77,11 +78,14 @@ def create_port_postcommit(self, context):
 
     def update_port_precommit(self, context):
         """update_port_precommit."""
-        if not self._is_port_of_interest(context):
```

Comment:
Hi Vivek,
I have not called is_port_of_interest() method in update_port_precommit & delete_port_precommit
because is_port_of_interest() checks binding_profile contents.

In update port/delete port  , still user can update/delete the neutron port with empty binding profile and hence is port of interest will fail the condition 
if vnic_type != portbindings.VNIC_BAREMETAL or not binding_profile:
            return False